### DISCUSSION TITLE
请问关于引擎运行根目录的问题？

### DISCUSSION THREAD

**Discussion by rocketman123456** [BoomingTech/Piccolo#225] [Q&A]
在Pilot引擎中，我看到引擎运行的目录是在CMake中定义来add_compile_definitions("PILOT_ROOT_DIR=${BINARY_ROOT_DIR}")来决定的。这个对于编译完的程序就没办法更改了，请问能不能通过设置特殊名字的根目录表示文件或者是采用相对路径的方法解决这个问题呢？

**Comment by Ol6rin** [comment]
你好，感谢对Pilot引擎的关注！这里我们后续会安排优化一下。
